### ISSUE TITLE
Silent crash on Big Sur

### ISSUE THREAD

**Issue by cikakosta** [vidalvanbergen/ViMediaManager#49]
**Steps to reproduce**

1. Have a mac with Big Sur installed (in mu case its 11.0.1)
2. Start an ViMediaManager (0.7alpha22)

**Expected**

- App should show the collection

**Actual**

- A popup with Feedback form appears
- When tap Cancel, another popup appears: "Due to an unexpected error, this application might have become unstable, do you want to continue using it?"

**Comment by vidalvanbergen** [comment]
I don't currently have access to Big Sur to test this on, but could you try the build below to see if it helps?

http://www.mediafire.com/file/ompn3s581a6re0b/ViMediaManager.zip/file

**Comment by cikakosta** [comment]
Hey Vidal,
This is super strange, I was able to run the build you provided with no crashes on start. I quit that and used the version I had previosly and that one didnt crash any more. Not sure hows that connected, but it appears everything is working fine now.
Also, couple of things I noticed:
When I use right click on the movie from the list on the left side of the window it shows the beachball for 10-15 seconds before it shows the context menu.
In Preferences / General / Database language, its missing flag for Serbian 👍

I didnt test this with folders with a lot of content, just tried with couple of movies and one tvshow.

Many thanks for swift response :)
